micro-service/src/App.js

```diff
@@ -1,8 +1,12 @@
+'use client';
 import React from 'react';
 import * as Sentry from '@sentry/react';
 import { FeatureFeed } from '@apollosproject/web-shared/embeds';
-import { AppProvider } from '@apollosproject/web-shared/providers';
-import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import {
+  AppProvider,
```

Comment:
Re-importing these avoids potential "different version of `react-router-dom`" issues